substrate/client/consensus/babe/src/tests.rs

```diff
@@ -423,6 +423,7 @@ async fn run_one_test(mutator: impl Fn(&mut TestHeader, Stage) + Send + Sync + '
 	.await;
 }
 
+#[cfg(ignore_flaky_test)] // https://github.com/paritytech/polkadot-sdk/issues/48
```

Comment:
```suggestion
#[ignore] // https://github.com/paritytech/polkadot-sdk/issues/48
```

This is way better than introducing a non-existing cfg item?

Reply:
But the CI runs with `--ignored`, so it ignores the `#[ignored]` attribute...

Reply:
Good point.